src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -474,3 +525,31 @@ def test_init_raises_driver_errors_for_invalid_arguments(resource_name, channels
             # sessions, so we make a driver call on channels and ensure that errors
             session.channels[channels].output_function = nidcpower.OutputFunction.DC_VOLTAGE
     assert e.value.code == expected_error_code
```

Comment:
Add this case to `test_init_raises_driver_errors_for_invalid_arguments`:
`'Dev1/0', None, False`

(Didn't see this case tested anywhere)

Reply:
Added coverage in cb25392bc1430e3e03e2d741363eebfea78a4403 and 7696d83dbb5fc0e6b79e694bac38807e0830f8a1